src/main/kotlin/app/revanced/patches/reddit/layout/toolbar/ToolBarButtonPatch.kt

```diff
@@ -17,7 +17,7 @@ import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
 
 @Patch(
     name = "Hide toolbar button",
-    description = "Adds an option to hide the \"r/place\" or Reddit recap button in the toolbar.",
+    description = "Adds an option to hide the r/place or Reddit recap button in the toolbar.",
```

Comment:
See [this PR](https://github.com/inotia00/revanced-integrations/pull/34) for the reasoning